lib/base/process.cpp

```diff
@@ -164,8 +164,8 @@ static Value ProcessSpawnImpl(struct msghdr *msgh, const Dictionary::Ptr& reques
 		(void)close(fds[2]);
 
 #ifdef HAVE_NICE
-		if (adjustPriority && nice(5) < 0)
-			Log(LogWarning, "base", "Failed to renice child process.");
+		if (adjustPriority)
+			nice(5);
```

Comment:
If nice() fails, shouldn't we still log a warning?

Reply:
I think the child process should not write into the log. This could result in some mutex problems. Please correct me, if I'm wrong.